Exercises/4-count-types.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = array => {
+  const hash = {};
```

Comment:
Rename to better identifier
```suggestion
  const result = {};
```
or `res` or `types` or `counters`